data-model/SDP-sample-records/script_sample1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/script/1",
+  "label": "Serto (script)",
+  "entity_type": "script",
+  "uniform_name": "Serto (script)",
+  "description": "Serto is a cursive Syriac script that developed throughout the 7th century (CE) and became the dominant script in West Syrian manuscripts.",
+  "authority_files": [],
+  "associated_dates": [{
+    "type": "temporal-coverage",
+    "date_human_readable": "post-7th c. - present",
+    "date_normalized": "0650/"
+  }],
+  "geographic_coverage": "Typically used in West Syrian manuscripts",
```

Comment:
I'm torn between just having a note type here. Or including it in the description. Geographic coverage is too narrow a term as it is geo-cultural boundaries more than strictly geographic. I also don't think we can get more machine-readable than a prose description, and it seems this is really just a user-facing field for brief introductory characerizations rather than to enable any kind of systematic LOD study